usermods/BME280_v2/usermod_bme280.h

```diff
@@ -186,6 +187,9 @@ class UsermodBME280 : public Usermod
   {
     if (i2c_scl<0 || i2c_sda<0) { enabled = false; sensorType = 0; return; }
     
+    settings.bme280Addr = i2cAddress;
+    bme = BME280I2C(settings);
```

Comment:
That is not correct way to initialise the sensor.
What this does is it uses default copy constructor and overwrites original BME object without de-initialising it first.

You should follow dynamic sensor start & stop as seen in other usermods.

Reply:
I figured out how to do this. This also fixes the issue with having to reboot. Now it correctly contacts the proper chip address and resets between config updates.